BitwardenShared/UI/Platform/Settings/Settings/AccountSecurity/DeleteAccount/DeleteAccountView.swift

```diff
@@ -59,6 +27,85 @@ struct DeleteAccountView: View {
                 store.send(.dismiss)
             }
         }
+        .task {
+            await store.perform(.loadData)
+        }
+    }
+
+    /// This section is shown when the account can be deleted.
```

Comment:
⛏️ As a general rule you should comment what the property/method does or in this case creates/builds as to how the callers treat such method/property. I mean whether this is shown or not is not a concern of the actual computed property but one of the caller, so no need to comment it here because it may change in the future and one shouldn't need to update the comment here as well.
I would instead just comment:
```suggestion
    /// This is the section to delete an account.
```